pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -4788,6 +4788,66 @@ status:
 	}
 }
 
+func TestStopSidecars_WithInjectedSidecarsNoTaskSpecSidecars(t *testing.T) {
+	taskRun := parse.MustParseTaskRun(t, `
```

Comment:
please replace this with `MustParseV1beta1TaskRun`

Reply:
done!